extension/src/lib/chromeEvented.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * GPII Chrome Extension for Google Chrome
+ *
+ * Copyright 2016 RtF-US
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this license.
+ *
+ * You may obtain a copy of the license at
+ * https://github.com/GPII/gpii-chrome-extension/blob/master/LICENSE.txt
+ */
+
+/* eslint-env node */
+/* global fluid, require */
+
+"use strict";
+
+var gpii = fluid.registerNamespace("gpii");
+var chrome = chrome || require("sinon-chrome");
```

Comment:
Repeating this global boilerplate mentioning a particular test library is a bit awkward. Add a TODO here to make use of the Infusion ContextAwareness API - http://docs.fluidproject.org/infusion/development/ContextAwareness.html - no need to do this for this pull request